modin/experimental/pandas/io.py

```diff
@@ -223,6 +223,63 @@ def _read(**kwargs) -> DataFrame:
 read_csv_glob = _make_parser_func(sep=",")
 
 
+def read_json_glob(
+    filepath_or_buffer: FilePathOrBuffer,
+    orient=None,
+    typ="frame",
+    dtype=None,
+    convert_axes=None,
+    convert_dates=True,
+    keep_default_dates=True,
+    numpy=False,
+    precise_float=False,
+    date_unit=None,
+    encoding=None,
+    encoding_errors="strict",
+    lines=False,
+    chunksize=None,
+    compression="infer",
+    nrows: Optional[int] = None,
+    storage_options: StorageOptions = None,
+):  # noqa: MD02
```

Comment:
Silence following error: `ERROR:modin.experimental.pandas.io.read_json_glob:226:MD02:Spelling error in line: 261, found: 'numpy', reference: 'NumPy'`.

`doc_checker` does not expect keywords to be used as parameter names. While this is rare, we can leave regexp unchanged.